### DISCUSSION TITLE
How to open a remote link when we use the command: sqllineage -g -f

### DISCUSSION THREAD

**Discussion by yiyedaren** [reata/sqllineage#549] [Q&A]
The SQLlineage is installed on a Linux server and I want to access it remotely on Windows,and i tried to replace the localhost to the  real ip ,but it not work.

How to open a remote link when we use the command: sqllineage -g -f test.sql ?

@localhost sql]$ sqllineage -g -f test.sql 
 * SQLLineage Running on http://localhost:5000/?f=test.sql

**Comment by reata** [comment]
Convert this to discussion as this is a how-to use question instead of feature request or bug report.

On Linux server, run sqllineage with `sqllineage -g -f test.sql -H 0.0.0.0` and access from Windows with `http://{linux_server_ip}:5000/?test.sql`
